Applications/ApplicationsLib/UncoupledProcessesTimeLoop.cpp

```diff
@@ -88,7 +88,7 @@ void UncoupledProcessesTimeLoop::setInitialConditions(
                 ode_sys.getMatrixSpecifications()));
 
         auto& x0 = *_process_solutions[pcs_idx];
-        pcs.setInitialConditions(x0);
+        pcs.setInitialConditions(t0, x0);
```

Comment:
I do not understand why it is necessary to give the initial condition a time. I thought initial condition means t = 0.